src/backend/Csrs.Api/Instrumentation/InstrumentationExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class InstrumentationExtensions
+{
+    /// <summary>
+    /// Add OpenTelemetry Metric and Tracing.
+    /// </summary>
+    /// <param name="builder"></param>
+    public static void AddInstrumentation(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddOpenTelemetryMetrics(options =>
+        {
+            //options.AddPrometheusExporter(_ => { });
+        });       
+
+        builder.Services.AddOpenTelemetryTracing(builder =>
+        {
+            builder
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("csrs-api"))
+                .AddAspNetCoreInstrumentation()
+                .AddHttpClientInstrumentation();
+           
+            // for now, just use Zipkin
+            builder.AddZipkinExporter(options => {
```

Comment:
For now, these are just demonstrating the options. In the future, we will add some custom configuration to determine what exporter to use.